java/com/puppetlabs/trapperkeeper/services/webserver/jetty10/utils/InternalSslContextFactory.java

```diff
@@ -15,7 +15,7 @@ public class InternalSslContextFactory extends SslContextFactory.Server {
     private static int maxTries = 25;
     private static int sleepInMillisecondsBetweenTries = 100;
     private static final Logger LOG =
-            Log.getLogger(InternalSslContextFactory.class);
+            LoggerFactory.getLogger(InternalSslContextFactory.class);
```

Comment:
This was a deprecated logging method, so I updated it to the non-deprecated version.